typespec-ccf/proposals.tsp

```diff
@@ -0,0 +1,160 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+
+import "./cose.tsp";
+import "./common.tsp";
+
+@doc("A single step in a proposed change to the service.")
+model Action {
+  @doc("The name of action to perform. This should match an action defined in the service's constitution, so that it can be invoked by the `apply` function of the constitution if the proposal is accepted.")
```

Comment:
```suggestion
  @doc("The name of the action to perform. This should match an action defined in the service's constitution, so that it can be invoked by the `apply` function of the constitution if the proposal is accepted.")
```